### PR TITLE
Legaldocs

### PR DESCRIPTION
**Pull request by andyjih**
## Change description

Legal docs

## Related issues

Does this PR fix a Github Issue?

N/A

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

>

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: public/sitemap.xml [hunk 1]
```diff
@@ -195,8 +195,18 @@
...
     >daily</changefreq><priority>0.6</priority></url><url><loc
+    >https://www.grouparoo.com/legal/data-processing-addendum</loc><changefreq
+    >daily</changefreq><priority>0.6</priority></url><url><loc
+    >https://www.grouparoo.com/legal/hosted-services-addendum</loc><changefreq
+    >daily</changefreq><priority>0.6</priority></url><url><loc
     >https://www.grouparoo.com/legal/privacy</loc><changefreq
     >daily</changefreq><priority>0.6</priority></url><url><loc
+    >https://www.grouparoo.com/legal/subprocessors</loc><changefreq
+    >daily</changefreq><priority>0.6</priority></url><url><loc
+    >https://www.grouparoo.com/legal/terms-20211013</loc><changefreq
```
Comment (evantahler) [inline public/sitemap.xml]: We probably don't want the old terms in the sitemap

File: pages/legal/terms-20211013.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import hydrate from "next-mdx-remote/hydrate";
+import { PageNavigation } from "../../components/pageNavigation";
+
+export default function Meet({ pageProps }) {
+  const content = hydrate(pageProps.source, {});
+
+  return (
+    <>
+      <Head>
+        <title>Grouparoo Master Terms and Conditions</title>
```
Comment (evantahler) [inline pages/legal/terms-20211013.tsx]: Probably should have the old date in the title

File: data/legal/data-processing-addendum.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+   3.2.9. For the purposes of Annex I, Section C (Competent Supervisory Authority), the competent supervisory authority identified in accordance with Clause 13 of the EU SCCs is the competent supervisory authority communicated by Customer to Grouparoo. If Customer does not communicate a competent supervisory authority to Grouparoo, the competent supervisory authority shall be the Irish Data Protection Commission.
+
+   3.2.10. For the purposes of Annex II, data importer has implemented and will maintain appropriate technical and organizational measures to protect the security, confidentiality and integrity of Customer Personal Data as described at Grouparoo’s [Security Page](/legal/security) or as otherwise made reasonably available by Customer to Grouparoo.
```
Comment (evantahler) [inline data/legal/data-processing-addendum.md:71]: So... there is no `/legal/security` page.  That's why the test is failing.

**Review by evantahler — CHANGES_REQUESTED** [review]

**Review by evantahler — CHANGES_REQUESTED** [review]
There's no way to get to the `/legal/data-processing-addendum` page from any link or the footer. 

Maybe we need a `/legal` root page with links to all the other legal docs?

File: data/legal/terms-20211013.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Grouparoo Master Terms and Conditions
```
Comment (bleonard) [inline data/legal/terms-20211013.md:1]: If this isn't an actual url, I think you might want to put it in `components/documents/terms-20211013.md`
Then you can load that from the main page.
It also wouldn't show up in the sitemap.

Comment (bleonard) [reply data/legal/terms-20211013.md:1]: actually maybe that's already kinda like that. nevermind

**Comment by andyjih** [comment]
> There's no way to get to the `/legal/data-processing-addendum` page from any link or the footer.
> 
> Maybe we need a `/legal` root page with links to all the other legal docs?

We will only link to the data processing addendum from order forms/contracts that we send to people.

**Review by bleonard — APPROVED** [review]
lgtm!
(legal is good to me)

File: data/legal/security.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Grouparoo Security
+
+> _Last Updated October 27, 2021_
+
+Grouparoo Security Policies are covered here:
+
+Coming soon
```
Comment (evantahler) [inline data/legal/security.md:7]: How is this page different from https://www.grouparoo.com/docs/support/security?

File: utils/sitemap.ts [hunk 6]
```diff
@@ -66,7 +66,11 @@ export async function getSitemapStream() {
     )
     .sort((a, b) => a.split("/").length - b.split("/").length);
   paths.forEach((p) => {
-    if (!p.includes("/whats-new/") && !p.includes("/trial_landing")) {
+    if (
+      !p.includes("/whats-new/") &&
+      !p.includes("/trial_landing") &&
+      !p.includes("/terms-")
```
Comment (evantahler) [inline utils/sitemap.ts]:
```suggestion
      !p.includes("/legal/terms-")
```

File: components/footer.tsx [hunk 7]
```diff
@@ -241,6 +241,9 @@ export default function Footer() {
                   <Link href="/legal/privacy">
                     <a>Privacy Policy</a>
                   </Link>
+                  <Link href="/legal/terms">
```
Comment (evantahler) [inline components/footer.tsx:245]: Missing a `<br />`

**Review by evantahler — CHANGES_REQUESTED** [review]

**Comment by evantahler** [comment]
Some other misc things to point out:

The address is code :D 
Screen Shot 2021-10-28 at 2 56 27 PM

The tables on this page should probably have some padding between each cell
Screen Shot 2021-10-28 at 2 57 38 PM

**Comment by evantahler** [comment]
Screen Shot 2021-10-28 at 2 57 51 PM

File: data/legal/data-processing-addendum.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
...ited Kingdom; (iii) all references to the “Commission” shall be deemed to refer to the Information Commissioner; (iv) all references to the “European Economic Area” or the “European Union” shall be deemed to refer to the United Kingdom; (v) for the purposes Appendix 1 to the UK SCCs, information about the exporter and importer, the categories of Data Subjects, types of Personal Data and type of Processing operations are as set out in Schedule 1 to this DPA; and (vi) for the purposes Appendix 2 to the UK SCCs, the security measures are as described at Grouparoo’s [Security Page](/legal/security)or as otherwise made reasonably available by data importer to the data exporter. The parties acknowledge that the Information Commissioner’s Office has not yet approved new standard contractual clauses under the UK GDPR. The UK SCCs will apply only until such time as the Information Commissioner’s Office issues new standard contractual clauses under the UK GDPR. Once approved, the parties shall work together, in good faith, to enter into an updated version of the UK SCCs or negotiate an alternative solution to enable transfers of Customer Personal Data in compliance with Data Protection Laws.
```
Comment (evantahler) [inline data/legal/data-processing-addendum.md]: > Appendix 2 to the UK SCCs, the security measures are as described at Grouparoo’s [Security Page](/legal/security)or as otherwi

Missing a space after the link

File: data/legal/security.md (hunk 5 above)
Comment (andyjih) [reply data/legal/security.md:7]: This page will cover our hosted security policies and practices. Will be updating this in the next couple weeks.

**Review by evantahler — APPROVED** [review]
👩‍⚖️

**Pull request merged**